patch/patch.py

```diff
@@ -0,0 +1,317 @@
+import argparse
+import os
+import shutil
+from common import (
+    check_path,
+    process_commit_id,
+    git_init,
+    save_patch_to_tmp,
+    check_branch_name,
+)
+
+path = os.getcwd()
+
+
+def _add_auto_generate_args():
+    """Set input argument."""
```

Comment:
the argparse can put in utils.py or new file argument.py?

Reply:
Because the argument is not too much , we didn't create a new utils.py or argument.py.